contracts/minter-suite/Minters/MinterSEAV0.sol

```diff
@@ -393,6 +397,27 @@ contract MinterSEAV0 is ReentrancyGuard, MinterBase, IFilteredMinterSEAV0 {
         emit MinterTimeBufferUpdated(_minterTimeBufferSeconds);
     }
 
+    /**
+     * @notice Sets the gas limit during ETH refunds when a collector is
+     * outbid. This value should be set to a value that is high enough to
+     * ensure that refunds are successful for commonly used wallets, but low
+     * enough to avoid excessive abuse of refund gas allowance during a new
+     * bid.
+     * @dev max gas limit is 63,535, which is considered a future-safe upper
+     * bound.
+     * @param _minterRefundGasLimit Gas limit to set for refunds. Must be between
+     * 5,000 and max uint16 (63,535).
+     */
+    function updateRefundGasLimit(uint16 _minterRefundGasLimit) external {
+        _onlyCoreAdminACL(this.updateRefundGasLimit.selector);
+        // CHECKS
+        // @dev max gas limit implicitly checked by using uint16 input arg
+        require(_minterRefundGasLimit >= 5000, "Only gte 5_000");
```

Comment:
Why is this 5k, vs. e.g. 21k?

Reply:
This is mostly to prevent a footgun when an admin is configuring, so was trying to just impose limits that seemed somewhat reasonable.

The `5_000` gas limit is a nice round number that should still support transferring to most smart contract wallets. Solidity's `.transfer` function imposes a gas limit of `2_300`, but as we know, that isn't enough gas for sending ETH to things like a gnosis safe after London hardfork. With `5_000` gas, transfers should still be successful to a gnosis safe, but receive logic would still be pretty limited. `21_000` seems like it would be quite a lot for a lower bound (although it might be an appropriate value for an admin to configure as the actual refund gas limit).

Reply:
Makes sense ser. 21k from being standard for EOA initiated transfers IIUC, but perhaps that is also bad limit here.

5k does feel somewhat random/arbitrary, so wonder if we should anchor lower (e.g. `2600` based on https://github.com/wolflo/evm-opcodes/blob/main/gas.md#aa-call-operations)  – where is `2300` from?

Reply:
@jakerockland  good point, 5k is too random. but 2600 is just the base cost of accessing the cold address, so things would fail there too.

I think at a bare-minimum, we would want to at least not remove support for gnosis safes.

Gnosis safes emit an event upon receiving ETH (ref https://github.com/safe-global/safe-contracts/blob/cb22537c89ea4187f4ad141ab2e1abf15b27416b/contracts/common/NativeCurrencyPaymentFallback.sol#L16)

Here is a transaction where someone sent our very own Gnosis safe 0.21 ETH in Nov '22: https://etherscan.io/tx/0x39fb07b63552d9c62b369060509f1b76b412311e7398e06ecf691481d8b2add7

That transaction used `27_329` gas. Subtracting off base tx `21_000` gas means `6_329` gas was used to access the cold address, execute the receive function, and emit the event. The gas used to emit an event is variable and can be based on the value sent, so I would recommend we set the minimum to something like `7_000` after looking at this example.

I've updated min limit to be 7_000, and added a dev comment for logic behind the number, in commit [a9b9af0](https://github.com/ArtBlocks/artblocks-contracts/pull/504/commits/a9b9af0be0fc7b5e36521d4a01b8738fc56bee66)

Reply:
Rad – that SGTM ser. Thank you for the thoughtful responses here!